src/pages/Shared/AddPostModal/AddPostModal.js

```diff
@@ -0,0 +1,137 @@
+import {
+  styled,
+  Modal,
+  Box,
+  Typography,
+  Grid,
+  Divider,
+  Avatar,
+  TextareaAutosize,
+  IconButton,
+  Tooltip,
+  Button,
+} from "@mui/material";
+import react, { useState } from "react";
+import { ImImage, ImPlay, ImFileEmpty } from "react-icons/im";
+import { MdOutlineClose } from "react-icons/md";
+
+const ModalBoxContainer = styled(Box)(({ theme }) => ({
+  position: "absolute",
+  top: "30%",
```

Comment:
Suggestion: try not to use big values in the top and left when using absolute it will affect in responsiveness